src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java

```diff
@@ -17,25 +29,188 @@
 @Transactional
 public class GameService {
 
-  public List<BlackCard> getCards() {
-    /*
-    This is a helper to get black cards
-    TODO: delete and replace this, as soon as the jpa black cards gets populated
+    private final GameRepository gameRepository;
+    private final PlayRepository playRepository;
+    private final WhiteCardRepository whiteCardRepository;
+    private final BlackCardRepository blackCardRepository;
+
+    private final SecureRandom rand = new SecureRandom();
+
+    @Autowired
+    public GameService(@Qualifier("gameRepository") GameRepository gameRepository,
+                       @Qualifier("playRepository") PlayRepository playRepository,
+                       @Qualifier("WhiteCardRepository") WhiteCardRepository whiteCardRepository,
+                       @Qualifier("BlackCardRepository") BlackCardRepository blackCardRepository) {
+        this.gameRepository = gameRepository;
+        this.playRepository = playRepository;
+        this.whiteCardRepository = whiteCardRepository;
+        this.blackCardRepository = blackCardRepository;
+    }
+
+  /**
+   * Fetch n random Cards from the blackCardRepository
+   * @return List of Black card
+   */
+  // See solution from https://stackoverflow.com/a/52409343/17532411
+  public List<BlackCard> getNRandomBlackCards(int numOfCards) {
```

Comment:
Consider changing this method to always return fixed amount of cards (if we always want them to be able to choose from the same amount of cards) Otherwise maybe create a variable with fixed amount of black cards per day

Reply:
Created a class variable for it, but seems a bit unnecessary because we use it only once.